os/booting-on-ikoula.md

```diff
@@ -0,0 +1,266 @@
+# Running CoreOS on Ikoula Cloud
+
+<a href="https://www.Ikoula.com">Ikoula</a> is a french cloud hosting company.
+Its CloudStack 4.5 based cloud is deployed on its own datacenters worldwide on 
+several Basic and Advanced networking zones. Currently zones are available on
+France, Germany, Netherlands, USA East, Singapore.
+
+Hosts deployed are all backed by fast SSD storage. Thanks to this and some
+network and OS optimizations, Ikoula is first in performance amongst several
+french cloud providers (<a href="http://www.journaldunet.com/solutions/cloud-computing/comparatif-cloud.shtml#cloud-comparatif-de-la-performance">source JDN</a>) 
+
+### Networking Zones on Ikoula Cloud
+
+On Basic networking zones, you have the support of security groups and a directly
+assigned public ip address as in many other cloud providers.
+
+On Advanced networking zones, you have a virtual router deployed which protects
+your own private vlan network. Instances deployed on this private vlan network
+receive a private ip address from a range you can choose, and they have external
+access through the virtual router. On Advanced networking zones you have access
+to VPC which can be connected to your own office network by means of using site to
+site VPNs.
+
+### CoreOS images on Ikoula Cloud
+
+You can deploy and boot officially built CoreOS images on Ikoula Cloud !
+
+You will need :
+
+- a valid account
+- a valid cloud public subscription
+
+Please note that images deployed on Ikoula Cloud update themselves automatically.
+The reboot strategy is "best effort", that means that if the deployed CoreOS is 
+standalone then it will reboot automatically. You can however [disable this behaviour][reboot-after-update].
+We also provide a FAQ on how to [achieve][coreos-update-manually] this.
+
+[reboot-after-update]: {{site.baseurl}}/docs/cluster-management/debugging/prevent-reboot-after-update
```

Comment:
side note: we need to go through all of the old docs and remove those links since we're rewriting them